docs/rule/attributes-order.md

```diff
@@ -0,0 +1,60 @@
+# attributes-order
+
+Element attributes must be ordered. By default, the order is:
+
+- arguments
+- attributes
+- element modifiers
+
+Note that this ordering only applies to angle bracket invocation syntax, as mustache/curly invocation syntax only includes a single type `attributes`.
+
+## Examples
+
+This rule **forbids** the following (in the default ordering):
+
+```hbs
+<MyComponent data-test-id="Hello" @name="World" />
+```
+
+```hbs
+<MyComponent {{did-render this.someAction}} @name="Hello" />
+```
+
+```hbs
+<MyComponent {{did-render this.someAction}} data-test-id="World" />
+```
+
+```hbs
+<MyComponent a="2" @b="1" />
+```
+
+This rule **allows** the following:
+
+```hbs
+<MyComponent @name="1" bar="baz" {{did-render this.someAction}} ...attributes aria-role="button" />
+<MyComponent @name="1" aria-role="button" />
+```
+
+```hbs
+<MyComponent @name="1" ...attributes />
+<MyComponent @name="1" {{did-render this.someAction}} />
+<MyComponent @name="1" bar="baz" />
+```
+
+## Configuration
+
+The following values are valid configuration:
+
+- boolean -- `true` to enable / `false` to disable
+- object -- An object with the following keys:
+  - `attributeOrder` -- An array of attribute types (defaults to `['arguments', 'attributes', 'modifiers']`
```

Comment:
Should we just call this option `order`? Is there any benefit or need to be specific about what ordering we're talking about it here? Or is the type of ordering implicit from the rule name so we could just call it `order`? What do you think?

Reply:
Simplifying to `order` makes sense given the rule name already gives the context.